success-criteria.md

```diff
@@ -443,39 +425,29 @@ stakeholder group.
 -   The publisher ad-funded business model is supported by addressing marketers
     needs and wants.
 
-    -   [Impacting these marketer interests, reduces publishers' revenue earned](https://services.google.com/fh/files/misc/disabling_third-party_cookies_publisher_revenue.pdf)
+    -   The capacity to positively match type and quality of publishers' content with advertising content
 
-    -   Negatively impacts the quantity and quality of publishers’ content
+    -   Supporting quantity and quality of publisher content to create a diverse and competitive field of ad opportunities, which supports diverse publishers.
 
-    -   Reducing quantity and quality of publishers’ content, conflicts with the
-        interests of an open society
-
--   Marketers who invest in cross-publisher advertising need scaled,
-    interoperable measurement and control:
+-   Marketers who invest in cross-publisher advertising need reporting, insights and recommendations to improve future business outcomes:
 
     -   Pre-campaign media planning and forecasting
 
-        -   Data enrichment services (e.g., context, geographic, technographic,
-            audience)
+        -   Services which provide additional dimensions to target, including general demographic definitions.
```

Comment:
This is poorly defined, to what extent do they need this? To what degree of precision? Proposed better phrasing.